### PR TITLE
Support multiple targets with the same selector but different valid anchors.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[Bug: With a single anchor-name, anchor element may be different for each given target element.](https://trello.com/c/MKApHxX7/83-bug-with-a-single-anchor-name-anchor-element-may-be-different-for-each-given-target-element)

File: src/validate.ts [hunk 1]
```diff
@@ -82,7 +78,11 @@ export async function isValidAnchorElement(
     }
     const lastInChain = anchorCBchain[anchorCBchain.length - 1];
 
-    if (lastInChain && isAbsolutelyPositioned(lastInChain as HTMLElement)) {
```
Comment (jgerigmeyer) [inline src/validate.ts:85]: @sanajaved7 Some new unit tests using jsdom were failing when `lastInChain` was `window`, so I updated this (and modified the `window` check).

File: src/polyfill.ts [hunk 2]
```diff
@@ -253,25 +249,34 @@ function position(rules: AnchorPositions) {
...
           });
+        } else {
+          // Use fallback value
+          const resolved = await getPixelValue({
+            targetProperty: property,
+            anchorSide: anchorValue.anchorSide,
+            anchorSize: anchorValue.anchorSize,
+            fallback: anchorValue.fallbackValue,
+          });
+          root.style.setProperty(anchorValue.uuid, resolved);
```
Comment (jgerigmeyer) [inline src/polyfill.ts:260]: Before, we weren't using the fallback value consistently for invalid queries. This fixes that.

**Review by sanajaved7 — APPROVED** [review]
This also looks good, thanks for catching and fixing this! 🐛

**Pull request merged**